frontend/util.lua

```diff
@@ -226,6 +226,8 @@ end
 ---- @treturn string hour string
 --- @note: The MS CRT doesn't support either %l & %k, or the - format modifier (as they're not technically C99 or POSIX).
 ---        They are otherwise supported on Linux, BSD & Bionic, so, just special-case Windows...
+---        We *could* arguably feed the os.date output to gsub("^0(%d)(.*)$", "%1%2"), but, while unlikely,
+---        it's conceavable that a translator wouldput something other that the hour at the front of the string ;).
```

Comment:
typo

Reply:
*Inconceivable*! :D

Reply:
(I solemnly swear not to have made the typo just to land that reference :D).

Reply:
Or unpossible? :-P